fake_joint_driver/src/fake_joint_driver.cpp

```diff
@@ -25,103 +24,145 @@ FakeJointDriver::FakeJointDriver(void)
   pnh.getParam("exclude_joints", exclude_joints_);
   pnh.getParam("start_position", start_position_map);
 
-  for (auto it=start_position_map.begin(); it!=start_position_map.end(); it++)
-  {
+  for (auto it = start_position_map.begin(); it != start_position_map.end();
+       it++) {
     ROS_DEBUG_STREAM("start_position: " << it->first << ": " << it->second);
   }
 
-  for (auto i=0; i<include_joints_.size(); i++)
-  {
+  for (auto i = 0; i < include_joints_.size(); i++) {
     ROS_DEBUG_STREAM("include_joint[" << i << "]" << include_joints_[i]);
   }
-  for (auto i=0; i<exclude_joints_.size(); i++)
-  {
+  for (auto i = 0; i < exclude_joints_.size(); i++) {
     ROS_DEBUG_STREAM("exclude_joint[" << i << "]" << exclude_joints_[i]);
   }
   // Read all joints in robot_description
-  if (use_description_)
-  {
+  if (use_description_) {
     urdf::Model urdf_model;
-    if (urdf_model.initParam("robot_description"))
-    {
-      for (auto it=urdf_model.joints_.begin(); it!=urdf_model.joints_.end(); it++)
-      {
+    if (urdf_model.initParam("robot_description")) {
+      for (auto it = urdf_model.joints_.begin(); it != urdf_model.joints_.end();
+           it++) {
         urdf::Joint joint = *it->second;
         // remove fixed and unknown joints
-        if (joint.type == urdf::Joint::FIXED || joint.type == urdf::Joint::UNKNOWN)
-        {
+        if (joint.type == urdf::Joint::FIXED ||
+            joint.type == urdf::Joint::UNKNOWN) {
           continue;
         }
         joint_set.insert(joint.name);
       }
-    }
-    else
-    {
-      ROS_WARN("We cannot find the parameter robot_description.");    
+    } else {
```

Comment:
Just curious, what version of clang_format did you run on this? In the future, it would be wise to run clang-format in a separate commit so it can easily be reverted, if needed.

Reply:
I used clang-format-10. Will take formatting into separate commit in the future.

Reply:
https://github.com/PickNikRobotics/fake_joint/pull/1/commits/9f101448b4d9e94dc2081a2c380528f3b50c65ce